src/a2a-net.Client.Http/Services/A2AProtocolHttpClient.cs

```diff
@@ -61,16 +61,25 @@ public virtual async IAsyncEnumerable<RpcResponse<TaskEvent>> SendTaskStreamingA
         using var content = new StringContent(JsonSerializer.Serialize(request), Encoding.UTF8, MediaTypeNames.Application.Json);
         using var httpRequest = new HttpRequestMessage(HttpMethod.Post, Options.Endpoint) { Content = content };
         httpRequest.EnableWebAssemblyStreamingResponse();
-        using var httpResponse = await HttpClient.SendAsync(httpRequest, cancellationToken).ConfigureAwait(false);
+        using var httpResponse = await HttpClient.SendAsync(httpRequest, HttpCompletionOption.ResponseHeadersRead, cancellationToken).ConfigureAwait(false);
         httpResponse.EnsureSuccessStatusCode();
-        var httpResponseStream = await httpResponse.Content.ReadAsStreamAsync(cancellationToken).ConfigureAwait(false);
+
         using var streamReader = new StreamReader(await httpResponse.Content.ReadAsStreamAsync(cancellationToken).ConfigureAwait(false));
         while (!streamReader.EndOfStream)
         {
             var sseMessage = await streamReader.ReadLineAsync(cancellationToken).ConfigureAwait(false);
             if (string.IsNullOrWhiteSpace(sseMessage)) continue;
             var json = sseMessage["data: ".Length..].Trim();
-            var e = JsonSerializer.Deserialize<RpcResponse<TaskEvent>>(json)!;
+            RpcResponse<TaskEvent>? e = null;
+            try
+            {
+                e = JsonSerializer.Deserialize<RpcResponse<TaskEvent>>(json)!;
+            }
+            catch (Exception ex)
```

Comment:
Could you remove variable, so that we address the `The variable 'ex' is declared but never used` warning?